### PR TITLE
SONAR-7417 all CE to run on more than 1 thread

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

File: server/sonar-server/src/test/java/org/sonar/server/computation/taskprocessor/CeProcessingSchedulerImplMultiThreadTest.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    CeProcessingSchedulerExecutorService processingExecutorService = null;
+    CeProcessingSchedulerImpl underTest = null;
+    try {
+      processingExecutorService = new CeProcessingSchedulerExecutorServiceImpl(ceConfiguration);
+      underTest = new CeProcessingSchedulerImpl(ceConfiguration, processingExecutorService, ceWorkerRunnable);
+
+      underTest.startScheduling();
+
+      // scheduling starts only after 10ms, leave 100ms for the tasks to run
+      Thread.sleep(100);
```
Comment (SonarTech) [inline server/sonar-server/src/test/java/org/sonar/server/computation/taskprocessor/CeProcessingSchedulerImplMultiThreadTest.java]: CRITICAL Remove this use of "Thread.sleep()". [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2925)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* CRITICAL 1 critical

Watch the comments in this conversation to review them.

**Comment by simonbrandhof** [comment]
The property "sonar.ce.workerCount" should be documented in sonar.properties.
Then LGTM.

**Comment by sns-seb** [comment]
`sonar.properties` completed

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* CRITICAL 1 critical

Watch the comments in this conversation to review them.

**Pull request merged**